src/i18n/translations.ts

```diff
@@ -250,10 +236,42 @@ export default {
         addLayerErrorMsg: 'Fehler beim Hinzufügen des Layers zur Karte.',
         loadLayerErrorMsg: 'Fehler beim Laden des Layers.',
         extentNotSupportedErrorMsg: 'Zoomen auf Gesamtansicht wird für diesen Typ nicht unterstützt.'
+      },
+      VerifyProviderDetailsField: {
+        title: 'Keine Informationen des Authentication-Providers verfügbar'
+      },
+      UploadLayerButton: {
+        success: {
+          message: 'Layer erfolgreich erstellt',
+          description: 'Datei {{fileName}} wurde erfolgreich geladen und der Layer {{layerName}} erstellt'
+        },
+        error: {
+          message: 'Konnte Layer nicht erstellen',
+          description: 'Fehler beim Hochladen der Datei {{fileName}}',
+          descriptionSize: 'Der Upload überschreitet das Limit von {{maxSize}} MB',
+          descriptionFormat: 'Der Dateityp ist nicht unterstützt ({{supportedFormats}})',
+          descriptionZipContent: 'Mehrere Geodatensätze innerhalb eines Archivs sind nicht unterstützt'
+        },
+        title: 'Layer erstellen'
+      },
+      CreateAllUsersButton: {
+        title: 'Nutzer synchronisieren',
+        tooltip: 'Add all missing users from the user provider',
+        success: 'Nutzer erfolgreich erstellt',
+        error: 'Fehler beim Erstellen der Nutzer'
+      },
+      CreateAllGroupsButton: {
+        title: 'Gruppen synchronisieren',
+        tooltip: 'Add all missing users from the user provider',
+        success: 'Gruppen erfolgreich erstellt',
+        error: 'Fehler beim Erstellen der Gruppen'
+      },
+      CreateAllRolesButton: {
+        title: 'Rollen synchronisieren',
+        tooltip: 'Add all missing users from the user provider',
```

Comment:
```suggestion
        tooltip: 'Add all missing roles from the role provider',
```

Reply:
by the way, shouldn't this be in German?

Reply:
Thanks, nice catch, you're absolutely right!

Fixed in 38d4309ef04c93a5bd9dfc5fc663e76cfa8a3399.